catalog/resourcemanagement/resourcemanagement-querymonitor-plugin/src/main/java/org/codice/ddf/resourcemanagement/query/plugin/QueryMonitorPluginImpl.java

```diff
@@ -127,8 +134,20 @@ public QueryResponse process(QueryResponse input)
                     "Cannot remove ActiveSearch from the ActiveSearch Map. QueryResponse received in QueryMonitorPluginImpl was null.");
             return input;
         }
-        removeActiveSearch((UUID) input.getRequest()
-                .getPropertyValue(SEARCH_ID));
+        if (!removeActiveSearch((UUID) input.getRequest()
+                .getPropertyValue(SEARCH_ID))) {
+            QueryResponseImpl queryResponse = new QueryResponseImpl(input.getRequest(),
+                    new ArrayList<>(),
+                    0);
+            queryResponse.closeResultQueue();
+            Set<ProcessingDetails> processingDetails =
+                    new HashSet<>(Arrays.asList(new ProcessingDetailsImpl(QueryMonitorPlugin.class.getCanonicalName(),
+                            new StopProcessingException("Query was cancelled by administrator"))));
+            queryResponse.setProcessingDetails(processingDetails);
+            queryResponse.getProcessingDetails();
```

Comment:
What's going on here? You get the processing details, but don't do anything with it? Why call the method at all?